packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -269,6 +276,45 @@ class AndroidCameraCameraX extends CameraPlatform {
     });
   }
 
+  /// Gets the minimum supported exposure offset for the selected camera in EV units.
+  @override
+  Future<double> getMinExposureOffset(int cameraId) async {
```

Comment:
Why pass in a cameraId if the value is going to be ignored?

Reply:
I added comments about it not being used. I honestly don't know what the idea was behind passing it to some functions was, even though it's not used. I assume it was included in case any platforms needed access to it? @bparrishMines any insight?

Reply:
I wasn't involved when this feature was added, but I assume that it was included for other platform implementations and to follow the same pattern as the other methods.